### ISSUE TITLE
How to use authentication with "Sicherheitsdatei"?

### ISSUE THREAD

**Issue by stevstrong** [raphaelm/python-fints#143]
Hi there,
I was wondering whether this lib supports authentication using "Sicherheitsdatei"?
If yes, how to use?
if not, can you give me some directions where should I start with implementation?
Many thanks in advance.

**Comment by raphaelm** [comment]
No, it does not. Basically all banks removed that feature as well a few years back IIRC, which bank is still doing this?

For implementing, start with reading the spec or better, finding the right part of the spec: https://www.hbci-zka.de/spec/3_0.htm

It's gonna be hard, probably.

**Comment by WolfgangFahl** [comment]
All Volksbanken seem to still have the feature.

**Comment by Heart1010** [comment]
> All Volksbanken seem to still have the feature.

No, only 'a few' of them still supports it - most Volksbanken have already removed it.

**Comment by raphaelm** [comment]
I thought that this feature was no longer possible for value to offer to consumers über PSD2 regulation that manages 2FA. But maybe it's still possible somehow and most banks just don't bother.

For business customers, banks usually recommend ebics.
